metadata/gclabels/gclabels.go

```diff
@@ -0,0 +1,47 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+// Labels that affect garbage collection.
+// Documented at https://github.com/containerd/containerd/blob/main/docs/garbage-collection.md
+package gclabels
+
+import "time"
+
+const (
+	LabelGCRoot       = "containerd.io/gc.root"         // Keep this object and anything it references.
+	LabelGCRef        = "containerd.io/gc.ref"          // This resource references another resource.
+	LabelGCRefSnap    = "containerd.io/gc.ref.snapshot" // This resource references an identified snapshot from a particular snapshotter.
```

Comment:
```suggestion
	LabelGCRefSnapshotPrefix    = "containerd.io/gc.ref.snapshot" // This resource references an identified snapshot from a particular snapshotter.
```

Reply:
"Prefix" on this one but not on the next one because the next one can be used standalone? I think the previous one would also get "Prefix" by that metric?

Reply:
Could you clarify the process behind this suggestion please? I want to rebase this but would rather get the naming right before I do.